backend/src/test/java/com/huellapositiva/integration/ResendEmailConfirmationActionShould.java

```diff
@@ -71,4 +72,25 @@ void update_hash_and_resend_email() {
         assertThat(initialHash, is(not(newHash)));
         verify(communicationService).sendRegistrationConfirmationEmail(any());
     }
+
+    @Test
+    void update_creation_date() throws Exception{
```

Comment:
This test should not exist, you can extend a test which already does resendEmailConfirmationAction.execute(); and assert the hash and the updatedOn time were actually updated